Src/BootCamp.Chapter/FileLogger.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class FileLogger : ILogger
+    {
+        private readonly string _logFileName;
+
+        public FileLogger()
+        {
+            _logFileName = CreateLogFile();
+            LogAction("Log File created");
+        }
+
+        public void LogAction(string action)
+        {
+            using (var writer = File.AppendText(_logFileName))
+            {
+                writer.WriteLine($"[{DateTime.Now}] {action}");
+            }
+        }
+
+        public void LogError(Exception error)
+        {
+            using (var writer = File.AppendText(_logFileName))
+            {
+                writer.WriteLine($"[{DateTime.Now}] {error}");
+            }
+        }
+
+        private string CreateLogFile()
+        {
+            Directory.CreateDirectory("Logs");
+            var currTime = DateTime.Now.ToString().Replace("/", "_").Replace(" ", "_").Replace(":", "_");
```

Comment:
Codacy found an issue: [Define the locale to be used in this string operation.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5742055)